rails-homework/app/models/appointment.rb

```diff
@@ -0,0 +1,4 @@
+class Appointment < ApplicationRecord
+  belongs_to :physician
+  belongs_to :patient
+end
```

Comment:
Please fix this GitHub alert. Each file should end with a NEWLINE symbol. You can configure your IDE to add newlines automatically. Fix it on all files from PR.

Reply:
I fixed.